src/definitions.h

```diff
@@ -15,6 +15,8 @@ static constexpr auto CLIENT_VERSION_STR = "13.10";
 static constexpr auto AUTHENTICATOR_DIGITS = 6U;
 static constexpr auto AUTHENTICATOR_PERIOD = 30U;
 
+#define OPENSSL_NO_DEPRECATED
```

Comment:
Why do we allow deprecated features?
All deprecated feature/API can be replaced by EVP_ API as far as i know

Reply:
We don't, this macro omits everything that is deprecated so it can't be used